sense/finetuning.py

```diff
@@ -305,6 +306,15 @@ def training_loops(net, train_loader, valid_loader, use_gpu, num_epochs, lr_sche
                 best_loss = valid_loss
                 best_state_dict = net.state_dict().copy()
 
+        # save the last checkpoint
+        model_state_dict = net.state_dict().copy()
+        model_state_dict = {clean_pipe_state_dict_key(key): value
+                            for key, value in model_state_dict.items()}
+        last_classifier = os.path.join(path_out, "checkpoints/", "last_classifier.checkpoint")
+        torch.save({
+            'model_state_dict': model_state_dict,
```

Comment:
you can directly do a torch.save(model_state_dict)
then We don t need to do: checkpoint_classifier['model_state_dict']

Reply:
> you can directly do a torch.save(model_state_dict)
> then We don t need to do: checkpoint_classifier['model_state_dict']

yes, I'll update it soon.